kornia/contrib/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .spatial_soft_argmax2d import SpatialSoftArgmax2d, spatial_soft_argmax2d
+from kornia.geometry.spatial_soft_argmax import SpatialSoftArgmax2d, spatial_soft_argmax2d
```

Comment:
why not relative path here ? at some point for the next release we should review the way we import/expose the api.

Reply:
Good question :) I have moved the function with PyCharm "refactor" tool, everything related was moved automatically. btw, I definitely can recommend PyCharm as an IDE